<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UWB - Coordinate reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="timeStepIntervalButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QProgressBar" name="progress">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="startButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="newFilesButton">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="serialComboBox"/>
    </item>
    <item row="0" column="5">
     <widget class="QCheckBox" name="serialCheckBox">
      <property name="text">
       <string>Use serial:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QComboBox" name="baudRate"/>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="baudrateLabel">
      <property name="text">
       <string>Baudrate:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="modeLabel">
      <property name="text">
       <string>Mode:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QComboBox" name="serialMode"/>
    </item>
    <item row="1" column="0" colspan="11">
     <widget class="QTextEdit" name="logger"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
